packages/airplane/internal/api/fetcher.ts

```diff
@@ -191,30 +190,13 @@ export class Fetcher {
         otel.propagation.inject(otel.context.active(), headers);
 
         try {
-          const response = await this.fetch(url.toString(), {
-            method,
-            body: body ? JSON.stringify(body) : undefined,
-            headers,
-          });
-          if (response.status >= 200 && response.status < 300) {
-            return (await response.json()) as Output;
-          }
-          throw await HTTPError.newFromResponse(method, response);
+          return makeRequest();
```

Comment:
Otherwise you leave the `try` clause too early.

```suggestion
          return await makeRequest();
```